README.rst

```diff
@@ -778,6 +780,13 @@ speed up further runs.
         ->setUsingCache(true)
     ;
 
+Exit codes
+----------
+* 0 OK
+* 1 No changes made
+* 4 Configuration error of the application
+* 8 Configuration error of a Fixer
```

Comment:
On 2.x line we already have:
```php
    const EXIT_STATUS_FLAG_HAS_INVALID_FILES = 4;
    const EXIT_STATUS_FLAG_HAS_CHANGED_FILES = 8;
```

For that reason, please, could you use 16 and 32 ?